apps/user-service/src/test/resources/secrets.yml

```
@@ -0,0 +1,5 @@
+secrets:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/test_oakcandb
+    username: postgres
+    password: 12345
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)